sycl/test/on-device/group_algorithms_sycl2020/permute_select.cpp

```diff
@@ -4,7 +4,7 @@
 // RUN: %ACC_RUN_PLACEHOLDER %t.out
 //
 // Missing __spirv_SubgroupId, __spirv_SubgroupMaxSize, __spirv_SubgroupShuffle* on AMD:
-// XFAIL: rocm_amd
+// XFAIL: hip_amd
```

Comment:
"on-device" tests were moved to llvm-test-suite repository, so these changes are not needed anymore.

Reply:
I have just rebased so all these should be gone now